### PR TITLE
Add a note about potential issues with ruby versioning

### PR DESCRIPTION
**Pull request by theothertomelliott**
This adds some explanation to the README to steer users away from the problem described in issue #38

### CODE REVIEW

**Comment by theothertomelliott** [comment]
It appears that the Travis tests fail on this PR because the GitHub access token secure environment variable is not available to pull requests.

See: http://docs.travis-ci.com/user/environment-variables/

File: README.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ In detail, the following tasks can be performed from a centralised location:
 
 [Local Git Repos]: http://bundler.io/v1.5/git.html
 
+## Ruby Version
+
+It is recommended to use system Ruby for CocoaPods development. If you are using rbenv or similar to manage your ruby environment, note that there are issues with certain dependencies under Ruby 2.2.x and higher.
```
Comment (kylef) [inline README.md]: I don't think we should recommend this, most of the CocoaPods team don't even use system Ruby. The CocoaPods app (recommended installation method for end users) uses a custom Ruby 2.2.x.

Comment (theothertomelliott) [inline README.md]: No problem, would the team be ok with a specific version being specified? The recommendation for system ruby was based on a comment from @segiddins in issue #38. 

I'm currently using 2.1.2 to get rake bootstrap to work. Is there an alternative workaround?

**Comment by segiddins** [comment]
Any ruby version > 2 should work

**Comment by theothertomelliott** [comment]
Based on comments, modified the message to remove specific recommendations.

**Comment by segiddins** [comment]
I've fixed this in CocoaPods proper -- thanks for the PR, though!

**Pull request closed without merging**